woocommerce/woocommerce-android
# Add toolbar for PaymentsHubFragment screen

Part of #10570 

### Description

This PR introduces individual toolbars for PaymentsHubFragment screen, replacing the shared toolbar previously used with MainActivity. This modification is essential to accommodate screens within a two-pane layout, ensuring that each screen maintains its distinct toolbar despite sharing a common parent.

### Testing instructions

Navigate to the Payment hub screen(Menu -> Payments) screen and ensure the toolbar and its action work as expected on both phone and tablet.
